packages/experiments/src/components/Nav/Nav.types.ts

```diff
@@ -36,31 +36,81 @@ export interface INavProps {
   navScrollerId?: string;
 
   /**
-   * Call to provide customized styling that will layer on top of the variant rules
+   * (Optional) Call to provide customized styling that will layer on top of the variant rules
    */
   getStyles?: IStyleFunction<INavStyleProps, INavStyles>;
 
   /**
-   * Used for telemetry
+   * (Optional) Used for telemetry
    */
   dataHint?: string;
 
+  /**
+   * (Optional) When enabled
+   * 1. Links will consider isHidden property to show/hide itself.
+   * 2. There will be a customization group with show more/less link to show/hide hidden links.
+   * 3. There will also be an edit nav link button. This is for the partner to implement the UX which
+   * will customize the isHidden property of the nav link (possibly through a flyout and refresh the Nav component).
+   */
+  enableCustomization?: boolean;
+
+  /**
+   * Used to override isHidden property of the Nav link when the "Show More" link is clicked
+   */
+  showMore?: boolean;
```

Comment:
why we need `showMore` in both state and props? I wonder if one should suffice?

Reply:
NavToggler controls the showMore state and passes on to Nav and SlimNav via props